base/array.jl

```diff
@@ -251,6 +251,14 @@ for (fname, felt) in ((:zeros,:zero), (:ones,:one))
         $fname(dims::Tuple) = ($fname)(Float64, dims)
         $fname(T::Type, dims...) = $fname(T, dims)
         $fname(dims...) = $fname(dims)
+
+        # #19635
+        function ($fname){T}(::Type{T}, arr::Array{T})
```

Comment:
I think this should be moved to `base/deprecated.jl`. That way it will be deleted when we delete the 0.6 deprecations (which happens late in the next release cycle); we might not notice/remember to delete it if it's in `array.jl`.